package.json

```diff
@@ -155,5 +162,8 @@
     "webpack-dev-server": "^4.11.1",
     "webpack-merge": "^5.0.9"
   },
-  "packageManager": "yarn@3.2.2"
+  "packageManager": "yarn@3.2.2",
+  "devDependencies": {
+    "@playwright/test": "^1.27.1"
```

Comment:
BTW: I mentioned on a zoom call once that there is [playwright-testing-library](https://github.com/testing-library/playwright-testing-library) akin to other libs we use like [react-testing-library](https://testing-library.com/docs/react-testing-library/intro/) (all part of the [Testing Library](https://github.com/testing-library) family).

I was initially using it, but decided to drop it. Playwright itself introduced some features inspired by testing-library, and the creator of playwright-testing-library says [^1]:

> Start out without playwright-testing-library and see how the new Testing Library API in 1.27 serves y

[^1]: https://github.com/testing-library/playwright-testing-library/issues/558#issuecomment-1274195339w